go.mod

```diff
@@ -5,6 +5,8 @@ go 1.18
 require (
 	github.com/caddyserver/certmagic v0.17.2
 	github.com/minio/minio-go/v7 v7.0.61
+	github.com/minio/pkg v1.7.5
```

Comment:
Actually there is a problem we can't link this package here since `minio/pkg` is AGPLv3